docs/index.html

```diff
@@ -0,0 +1,229 @@
+      <div
+        class="container mx-auto flex flex-wrap p-5 flex-col md:flex-row items-center"
+      >
+        <nav
+          class="flex lg:w-2/5 flex-wrap items-center text-base md:ml-auto text-transparent bg-clip-text bg-gradient-to-br from-green-400 to-green-600"
+        >
+          <a class="mr-5 hover:text-green-500" href="index.html">Home</a>
+
+          <a class="mr-5 hover:text-green-500" href="#features">Features</a>
+          <a class="mr-5 hover:text-green-500" href="#contributing"
+            >Contribute</a
+          >
+
+          <a class="hover:text-green-500" href="#support">Support</a>
+        </nav>
+        <a
+          class="flex order-first lg:order-none lg:w-1/5 title-font font-medium items-center text-white lg:items-center lg:justify-center mb-4 md:mb-0"
+          href="index.html"
+        >
+          <img
+            class="rounded-full"
+            src="img/obsidian_logo.png"
+            alt="obsidian_logo"
+            width="50px"
+            height="50px"
+          />
+          <span
+            class="ml-3 text-xl xl:block lg:hidden font-bold text-transparent bg-clip-text bg-gradient-to-br from-green-400 to-green-600"
+            >Obsidian</span
+          >
+        </a>
+        <div class="lg:w-2/5 inline-flex lg:justify-end ml-5 lg:ml-0">
+          <button
+            class="bg-gradient-to-r from-green-400 to-green-600 border-0 py-2 px-8 focus:outline-none hover:from-cyan-500 hover:to-blue-500 text-white font-bold py-2 px-4 rounded-full"
+          >
+            Invite Now →
+          </button>
+        </div>
+      </div>
+    </header>
+    <section
+      class="text-gray-400 bg-gradient-to-br from-gray-900 to-gray-800 body-font font-sans"
+    >
+      <div
+        class="container mx-auto flex px-5 py-24 items-center justify-center flex-col"
+      >
+        <img
+          class="mb-10 object-cover object-center rounded-full"
+          alt="hero"
+          src="Img/obsidian_logo.png"
+          width="200px"
+          height="200px"
+        />
+        <div class="text-center lg:w-2/3 w-full">
+          <h1
+            class="title-font font-bold sm:text-4xl text-3xl mb-4 font-medium text-green-500"
+          >
+            Obsidian: The multipurpose Discord Bot.
+          </h1>
+          <p class="leading-relaxed mb-8">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dicta
+            distinctio natus assumenda error! Voluptate, labore reprehenderit
+            similique repellat quam optio neque provident quod nesciunt ab
+            deleniti ad beatae corrupti quae! Lorem ipsum dolor sit amet
+            consectetur adipisicing elit. Rerum ipsum, nesciunt, labore sapiente
+            numquam minus ad illo ea id in perspiciatis mollitia corporis nam
+            quisquam impedit quasi. Aspernatur, dolore magnam.
+          </p>
+          <div class="flex justify-center">
+            <button
```

Comment:
Instead of wrapping a `<a>` inside a `<button>` you can just do
```html
<a href="#features"
  class="ml-4 inline-flex text-white bg-green-500 border-0 py-2 px-6 focus:outline-none hover:bg-green-600 rounded text-lg"
>See Features
</a>
```

This applies for all other instances of the same